include/roboptim/core/manifold-map/decorator/descriptive-wrapper.hh

```diff
@@ -100,15 +100,15 @@ namespace roboptim
   operator<<(std::ostream& o, DescriptiveWrapper<U, V>& descWrap)
   {
     o <<
-    "Displaying info about function in DescriptiveWrapper :" << "\n" <<
-    "the function is " << descWrap.fct().getName() << "\n" <<
-    "the function input size is " << descWrap.fct().inputSize() << "\n" <<
-    "the function output size is " << descWrap.fct().outputSize() << "\n" <<
-    "Displaying info about the function manifold in DescriptiveWrapper :" << "\n" <<
-    "the manifold is " << descWrap.manifold().name() << "\n" <<
-    "Is the manifold elementary ? : " <<
-    (descWrap.manifold().isElementary() ? "yes" : "no" ) << "\n" <<
-    "the manifold dimension is " << descWrap.manifold().representationDim() << "\n";
+      "Displaying info about function in DescriptiveWrapper :" << "\n" <<
+      "the function is " << descWrap.fct().getName() << "\n" <<
+      "the function input size is " << descWrap.fct().inputSize() << "\n" <<
+      "the function output size is " << descWrap.fct().outputSize() << "\n" <<
+      "Displaying info about the function manifold in DescriptiveWrapper :" << "\n" <<
+      "the manifold is " << descWrap.manifold().name() << "\n" <<
+      "Is the manifold elementary ? : " <<
+      (descWrap.manifold().isElementary() ? "yes" : "no" ) << "\n" <<
+      "the manifold dimension is " << descWrap.manifold().representationDim() << "\n";
```

Comment:
Note (even though that's not the focus of this PR): for better printing, RobOptim provides some handy [indentation support](https://github.com/roboptim/roboptim-core/blob/master/tests/indent.cc). Also, to be even pickier, whitespace before punctuation is [not valid in English](http://english.stackexchange.com/a/4646/59744).